### ISSUE TITLE
VoiceOver support for message box does not work properly

### ISSUE THREAD

**Issue by esporysz** [SAP/fundamental-ngx#12766]
#### Is this a bug, enhancement, or feature request?

bug

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)

angular: 18.2.4
fundamental: 0.53.1-rc.23

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;

1. turn on VoiceOver (MacOs version: 14.7)
2. Using only VoiceOver open MessageBox (you can try to open example from [documentation page](https://sap.github.io/fundamental-ngx/#/core/message-box#component-based)

After opening the MessageBox VoiceOver reads button from the footer at first. In this example it is 'Ok'. Voice Over should read header of the dialog, then its content and after that button from the footer.

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations.

Yes, accessibility expert comment "_Agree, in a simple reading mode it should introduce the component on that user is currently on_".

**Comment by dreamweiver** [comment]
@droshev , we are facing a similar issue and we got a Customer Ticket raised for the same.

so is there a way to prioritize this issue  on your end ?
